lib/internal/test_runner/runner.js

```diff
@@ -101,7 +102,10 @@ const kFilterArgValues = ['--test-reporter', '--test-reporter-destination'];
 const kDiagnosticsFilterArgs = ['tests', 'suites', 'pass', 'fail', 'cancelled', 'skipped', 'todo', 'duration_ms'];
 
 const kCanceledTests = new SafeSet()
-  .add(kCancelledByParent).add(kAborted).add(kTestTimeoutFailure);
+  .add(kCancelledByParent)
+  .add(kAborted)
+  .add(kTestTimeoutFailure)
+  .add(kTestBailedOut);
```

Comment:
I know you just added to it here, but can we initialize the set without repeatedly calling `add()`.

Reply:
done!